ci-operator/step-registry/telco5g/cnf-tests/telco5g-cnf-tests-commands.sh

```diff
@@ -338,4 +338,30 @@ echo -e "${err_msg}\n"
 if [ -f "summary.txt" ]; then
     cat summary.txt
 fi
+
+# Create a HTML report
+for feature in ${FEATURES}; do
+    xml_f="${ARTIFACT_DIR}/${feature}/cnftests-junit.xml"
+    if [[ -f $xml_f ]]; then
+        cp $xml_f ${ARTIFACT_DIR}/cnftests-junit_${feature}.xml
+    fi
+    xml_v="${ARTIFACT_DIR}/${feature}/validation_junit.xml"
+    if [[ -f $xml_v ]]; then
+        cp $xml_v ${ARTIFACT_DIR}/validation_junit_${feature}.xml
+    fi
+    xml_s="${ARTIFACT_DIR}/${feature}/setup_junit.xml"
+    if [[ -f $xml_s ]]; then
+        cp $xml_s ${ARTIFACT_DIR}/setup_junit_${feature}.xml
+    fi
+done
+
+python3 -m venv ${ARTIFACT_DIR}/myenv
+source ${ARTIFACT_DIR}/myenv/bin/activate
+git clone https://github.com/sshnaidm/html4junit.git ${ARTIFACT_DIR}/html4junit
```

Comment:
can u clone it somewhere else (outside of ${ARTIFACT_DIR} ? 
I dont think we need to keep the html4junit as an artifact..

Reply:
I can try shared dir.